RadixWallet/Features/ImportMnemonic/ImportMnemonic+View.swift

```diff
@@ -113,30 +125,71 @@ extension ImportMnemonic {
 								.padding(.bottom, .large3)
 						}
 
-						wordsGrid(with: viewStore)
-							.padding(.horizontal, .medium2)
-							.padding(.bottom, .large3)
-
 						if !viewStore.isWordCountFixed {
-							changeWordCountButtons(with: viewStore)
-								.padding(.horizontal, .medium2)
-								.padding(.bottom, .large3)
-						}
-
-						if viewStore.isShowingPassphrase {
-							passphrase(with: viewStore)
-								.padding(.horizontal, .medium2)
-								.padding(.bottom, .medium2)
+							VStack(alignment: .center) {
+								let label = "Number of Seed Phrase Words"
+								Text("\(label)").textStyle(.body1HighImportance).foregroundStyle(Color.app.gray1)
+
+								Picker(label, selection: viewStore.binding(
+									get: \.wordCount,
+									send: { .changedWordCountTo($0) }
+								)) {
+									ForEach(BIP39.WordCount.allCases, id: \.self) { wordCount in
+										Text("\(wordCount.rawValue)")
+											.textStyle(.body1Regular)
+									}
+								}
+								.pickerStyle(.segmented)
+							}
+							.padding(.horizontal, .large3)
+							.padding(.bottom, .medium2)
 						}
 
 						#if DEBUG
 						if viewStore.isReadonlyMode {
 							Button("DEBUG ONLY Copy") {
 								viewStore.send(.debugCopyMnemonic)
 							}
-							.buttonStyle(.secondaryRectangular(isDestructive: true))
-							.padding(.bottom, .medium1)
+							.buttonStyle(
+								.secondaryRectangular(
+									shouldExpand: true,
+									isDestructive: true,
+									isInToolbar: true
+								)
+							)
+							.padding(.horizontal, .medium2)
+							.padding(.bottom, .medium3)
 						} else {
+							Button("DEBUG Acc.Rec.Scan.mne (24)") {
+								viewStore.send(.debugUseTestingMnemonicWithActiveAccounts)
+							}
+							.buttonStyle(
+								.secondaryRectangular(
+									shouldExpand: true,
+									isDestructive: true,
+									isInToolbar: true
+								)
+							)
+							.padding(.horizontal, .medium2)
+							.padding(.bottom, .medium3)
+
+							Button(
+								"DEBUG zoo..vote (24)"
+							) {
+								viewStore.send(
+									.debugUseTestingMnemonicZooVote
```

Comment:
@kugel3 use this button for a mnemonic which has an active account at index `125` (thus testing we correctly display "None found" gray box for the first two batches